lib/renderer/win32-queryendsession-setup.ts

```diff
@@ -0,0 +1,8 @@
+import { ipcRenderer } from 'electron';
+
+export function setup (binding: typeof process['_linkedBinding']) {
+  // TODO(codebytere): fix typedef here.
+  (binding as any).setShutdownHandler(() => {
```

Comment:
The shutdown handler in renderer returns `undefined`, which would then always cancel the shutdown.